docs/components/input-stepper/overview.md

```diff
@@ -4,16 +4,16 @@ A web component that enables the user to increase and decrease a numeric value b
 
 ```js script
 import { html } from '@mdjs/mdjs-preview';
+import { loadDefaultFeedbackMessages } from '@lion/ui/validate-messages.js';
```

Comment:
Maybe we can move these to some global demo setups in our mdjs-viewer in the future